core/src/main/scala/scalan/arrays/ArrayOps.scala

```diff
@@ -380,10 +383,7 @@ trait ArrayOpsStd extends ArrayOps {
 }
 
 trait ArrayOpsExp extends ArrayOps with BaseExp { self: ScalanExp =>
-  def withElemOfArray[T, R](xs: Arr[T])(block: Elem[T] => R): R =
-    withElemOf(xs) { eTArr =>
-      block(eTArr.eItem)
-    }
+  def withElemOfArray[T, R](xs: Arr[T])(block: Elem[T] => R): R = withElemOf(xs) { eTArr => block(eTArr.eItem) }
```

Comment:
In current Idea version, when you set breakpoint on this line, it'll ask if the breakpoint is for the lambda or the entire line. One the other hand, I agree there is no point in making this change.